docs/en/ingest-management/fleet/fleet-limitations.asciidoc

```diff
@@ -31,5 +30,5 @@ single {kib} instance can handle:
 |1gb| 2000
 |===
 
-Beta releases are not officially supported, but we encourage you to
+Beta releases are not supported at a production level, but we encourage you to
```

Comment:
This line refers to paid Elastic Support, right? Are we providing support for Agent and Fleet now?

Reply:
I think you're trying to encourage users to adopt Fleet with this change, but I want to ensure we're not over-promising what our paid support will help with.

Reply:
We support beta versions at Sev3 or lower. We only support higher severity for GA features.

Reply:
👍 👍